Makefile

```diff
@@ -237,7 +241,7 @@ build-collector build-collector-debug:
 
 .PHONY: build-ingester build-ingester-debug
 build-ingester build-ingester-debug:
-	$(GOBUILD) $(DISABLE_OPTIMIZATIONS) -o ./cmd/ingester/ingester$(SUFFIX)-$(GOOS)-$(GOARCH) $(BUILD_INFO) ./cmd/ingester/main.go
+	$(GOBUILD) $(DISABLE_OPTIMIZATIONS) -o ./cmd/ingester/ingester$(SUFFIX)-$(GOOS)-$(GOARCH) ./cmd/ingester/main.go
```

Comment:
why did you drop `$(BUILD_INFO)` here?

Reply:
Ah good catch. It definitely should be there and deleting it was probably an accident.